.github/workflows/ci.yml

```diff
@@ -11,15 +11,13 @@ on:
 env:
   # Common versions
   GO_VERSION: '1.19'
-  GOLANGCI_VERSION: 'v1.50'
-  DOCKER_BUILDX_VERSION: 'v0.9.1'
+  GOLANGCI_VERSION: 'v1.50.0'
```

Comment:
I think there may be a couple of newer releases out if you want to go ahead and bump to latest 👍🏻

Reply:
Sure. But it should be same with the one we use in Makefile https://github.com/upbound/build/pull/229

Reply:
Agreed 👍🏻 can also override in Makefile in this repo